cves/CVE-2015-0219.yml

```diff
@@ -49,7 +49,16 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulnerability in Django caused underscore characters to be incorrectly
+  handled in certain headers in the system. The vulnerability allowed attackers
+  the ability to spoof headers in certain environments by using an underscore
+  instead of a dash. If a header was used in a way that was security-sensitive
```

Comment:
"If a header was used in a way that was security-sensitive way" consider removing extra "way" at end